packages/esm-ward-app/src/ward-patient-workspace/ward-patient.workspace.tsx

```diff
@@ -55,14 +36,16 @@ function WardPatientWorkspaceView({ patient }: WardPatientWorkspaceViewProps) {
       </div>
     </>
   );
-}
+};
+
+const PatientWorkspaceTitle: React.FC<WardPatientWorkspaceViewProps> = ({ patient }) => {
+  const { t } = useTranslation();
 
-function PatientWorkspaceTitle({ patient }: { patient: fhir.Patient }) {
   return (
     <>
-      <div>{getPatientName(patient)} &nbsp;</div>
-      <div className={styles.headerPatientDetail}>&middot; &nbsp; {patient.gender}</div>
-      <div className={styles.headerPatientDetail}>&middot; &nbsp; {age(patient.birthDate)}</div>
+      <div>{patient.person.display} &nbsp;</div>
```

Comment:
Are we swtiching back to display here because it applies the name template formatting and getPatientName does not?

Reply:
I changed this workspace to not use `usePatient()` because we already have the `Patient` object and don't need another fetch. `usePatient()` actually returns `fhir.Patient`, so I need to rework some of this since the fields are a bit different between those 2 types. `Patient.display` should respect the name template formatting defined server-side, and `getPatientName` is only used for `fhir.Patient`.